optuna/storages/_rdb/models.py

```diff
@@ -172,20 +160,6 @@ class TrialModel(BaseModel):
         StudyModel, backref=orm.backref("trials", cascade="all, delete-orphan")
     )
 
-    @classmethod
-    def find_by_id(
```

Comment:
We can remove this function after #2061 is merged.

But, other modification than here seems good to me!